core/server/data/meta/asset_url.js

```diff
@@ -39,11 +39,13 @@ function getAssetUrl(path, isAdmin, minify) {
         // we don't want to concat the path with our favicon url
         output += path;
 
-        if (!config.get('assetHash')) {
-            config.set('assetHash', utils.generateAssetHash());
-        }
+        if (config.get('env') === 'production') {
```

Comment:
AFAIK this still very much causes browser cache issues - unless the hashes are being changed on every single build, which seems to defeat the purpose ? I don't know why we need this in dev

I'm more than happy to be wrong though - if someone has better info